test/integration/rules/label-content-name-mismatch/label-content-name-mismatch.html

```diff
@@ -35,6 +35,11 @@
 <button id="incomplete9" aria-label="help">?</button>
 <button id="incomplete10" aria-label="&#x1F354;">&#x1F354;</button>
 <button id="incomplete11" aria-label="close">&#10060;</button>
+<button id="incomplete12" aria-label="close">×</button>
```

Comment:
These integration test updates look good, but could you please also update the unit tests at `/test/commons/text/is-human-interpretable.js`? (the unit tests would be a good place to add a test that motivates  the new `str.trim()` line you added)

Reply:
Addressed in several recent commits, several new unit tests and some improved integration tests have been added thanks!